packages/terra-alert/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated focus shift to rendered `alert` notification banners content that include an interactable action element and/or dismiss.
```

Comment:
Can we reword it to something like this?
```suggestion
  * Updated focus shift for `alert` notification banners to include notification banners that have an interactable action element and/or dismiss.
```

Reply:
Updated changelog here https://github.com/cerner/terra-core/pull/3823/commits/d4700b8f71c7da3a79709a02b3651a0419dcdd71#diff-b62ab7ae03f56c47f8aba033b108058b187c6bb44ff7dbe93a72c454c1393279